openra/tests/test_commands.py

```diff
@@ -36,6 +36,7 @@ def oramap_file_exists_for_map_id(self, map_id, data_fs: FS = Provide[Container.
         return False
 
     def test_it_creates_a_super_user_with_the_details_provided(self):
+        return
```

Comment:
intentional?

Reply:
Yes and no. I commented them out (well, early returned) as the tests aren't that great as then need to work with your actual FS etc. Once the views are updated to use those classes I've added, that wont be the case and they can just run like the others.

But they are still useful initially to make sure things are set up right, when setting up an environment.

So basically I don't think they should be deleted as they can be useful, but I don't think we really need them running every time someone tests. So I wonder if it's worth actually keeping this early return in for now?

Reply:
It would be good to include a comment here explaining this!

Reply:
Ok I've added a comment now